chain/client/src/info.rs

```diff
@@ -52,10 +51,8 @@ impl InfoHelper {
         }
     }
 
-    pub fn block_processed(&mut self, gas_used: Gas, gas_limit: Gas) {
+    pub fn block_processed(&mut self) {
         self.num_blocks_processed += 1;
-        self.gas_used += gas_used;
```

Comment:
well, now that you removed this - there will be 0 info about how much tx/gas processed.